arrow/arrow.py

```diff
@@ -969,8 +969,50 @@ def humanize(
                 )
             )
 
-    # query functions
+    def dehumanize(self, humanized_str="now", locale="en_us"):
+        """reads a localized, humanized representation of a relative difference in time,
+        returns a metadata of time.
+
+        :param humanized_str: (optional), a hunmanized string like "now" "in 11 seconds" "11 seconds ago"
+            Defaults to now in the current :class:`Arrow <arrow.arrow.Arrow>` object's timezone. Defaults to 'now'.
+        :param locale: (optional) a ``str`` specifying a locale.  Defaults to 'en_us'.
 
+        Usage::
+
+            >>> utc = arrow.utcnow()
+
+            >>> utc.dehumanize()
+            <Arrow [2013-05-09T03:49:12.311072+00:00]>
+
+            >>> utc.dehumanize('2 hours ago')
+            <Arrow [2013-05-09T01:49:12.311072+00:00]>
+        """
+        given_arrow = Arrow(
```

Comment:
You can directly use `self` because we are calling this function on an already existing instance of an arrow object.